src/main/res/values/strings.xml

```diff
@@ -166,4 +166,10 @@
     <string name="select_code_from_sd_card">Select code from SD Card</string>
     <string name="scan_code_from_another_device">Scan code from another device</string>
     <string name="please_wait">Please wait</string>
+    <string name="view_totals">View Totals</string>
+    <string name="interview_done">Interview Done</string>
+    <string name="interview_not_done">Interview Not Done</string>
+    <string name="interview_deffered">Interview Later</string>
+    <string name="interview_refused">Interview Refused</string>
+    <string name="interview_incomplete">Interview Incomplete</string>
```

Comment:
Plan to have these translated. Kindly create an issue to keep track of this

Reply:
@tychus6 could you create a separate issue for translating the above strings.

Reply:
@syedowaisali90 I still need the issue tracking translations. Kindly add a link here when it's created

Reply:
the translation issue is being tracked here https://github.com/onaio/WHOSteps/issues/19

Reply:
Confirmed